tempto-core/src/main/java/io/trino/tempto/query/JdbcQueryExecutor.java

```diff
@@ -138,7 +138,7 @@ private QueryResult executeQueryNoParams(String sql)
         }
     }
 
-    // TODO - remove this method as soon as Presto supports prepared statements
+    // TODO - remove this method as soon as Trino supports prepared statements
```

Comment:
Which it does. I am not sure however how having prepared statements make this method obsolete.